service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/HttpProxyNegativeCasesIT.java

```diff
@@ -50,12 +64,16 @@
     }};
 
     @Rule
-    public TestRule chain = createRuleChain(gateway, robot);
+    public TestRule chain =
+            RuleChain.outerRule(new MethodExecutionTrace()).around(robot).around(checkLogMessageRule).around(gateway);
 
     @Specification("http.proxy.http.1.0.request")
     @Test
     public void sendHttp_1_0_Request() throws Exception {
         robot.finish();
+        expectedPatterns = new ArrayList<String>(Arrays.asList(new String[] {
+                "http.proxy service .*. received an HTTP 1.0 request. HTTP 1.0 is not explicitly supported."
```

Comment:
The change in the comment above would require both explicitly stating what is currently ".*" in this regex.  And adding a .name("name") to the service element